content/SCALE/SCALEUIReference/SystemSettings/Services/SMBServicesScreen.md

```diff
@@ -45,22 +45,22 @@ Click **Save** or **Cancel** to close the configuration screen and return to the
 | Setting | Description |
 |---------|-------------|
 | **UNIX Charset** | Select the character set to use internally from the dropdown list of options. **UTF-8** is standard for most systems as it supports all characters in all languages. |
+| **Transport Encryption Behavior** | Select the option for the level of transport encryption to implement. Options and behaviors:<br><li>**Default - follow upstream/TrueNAS default** <br><li>**Negotiate - only encrypt transport if explicitly requested by the SMB client** <br><li>**Desired - encrypt transport if supported by client during session negotiation** <br><li>**Required - always encrypt transport (rejecting access if client does not support encryption - incompatible with SMB1 server `enable_smb1`)** </li>the TrueNAS and Samba default behavior allows SMB clients to negotiate different encryption levels for SMB shares. There is no technical limitation preventing an SMB client from negotiating an encrypted session if it is required. **Default** enables negotiating encryption but does not turn on data encryption globally per share. For more information on SMB1 and SMB2 session or per-share encryption, see [Samba Server SMB Encrypt(s)](https://www.samba.org/samba/docs/current/man-html/smb.conf.5.html#SERVERSMBENCRYPT). For more information on using Windows client-side SMB signing, see [Windows SMB Signing Policies](https://learn.microsoft.com/en-us/troubleshoot/windows-server/networking/overview-server-message-block-signing#policy-locations-for-smb-signing) for more information. |
```

Comment:
```suggestion
| **Transport Encryption Behavior** | Select the option for the level of transport encryption to implement. Options and behaviors:<br><li>**Default - follow upstream/TrueNAS default** <br><li>**Negotiate - only encrypt transport if explicitly requested by the SMB client** <br><li>**Desired - encrypt transport if supported by client during session negotiation** <br><li>**Required - always encrypt transport (rejecting access if client does not support encryption - incompatible with SMB1 server `enable_smb1`)** </li> The TrueNAS and Samba default behavior allows SMB clients to negotiate different encryption levels for SMB shares. There is no technical limitation preventing an SMB client from negotiating an encrypted session if it is required. **Default** enables negotiating encryption but does not turn on data encryption globally per share. For more information on SMB1 and SMB2 session or per-share encryption, see [Samba Server SMB Encrypt(s)](https://www.samba.org/samba/docs/current/man-html/smb.conf.5.html#SERVERSMBENCRYPT). For more information on using Windows client-side SMB signing, see [Windows SMB Signing Policies](https://learn.microsoft.com/en-us/troubleshoot/windows-server/networking/overview-server-message-block-signing#policy-locations-for-smb-signing). |
```

Reply:
Also whatever change made to the "There is no technical limitation" on the tutorial should be reflected here as well

Reply:
changed description in the next commit.